### ISSUE TITLE
Global search not working - No results/boxes shown

### ISSUE THREAD

**Issue by moostrich** [sonata-project/SonataAdminBundle#7311]
#### Sonata packages

<details><summary>show</summary>
<p>

```
$ composer show --latest 'sonata-project/*'
sonata-project/admin-bundle              dev-master a81f52f dev-master a81f52f The missing Symfony Admin Generator
sonata-project/block-bundle              4.6.0              4.6.0              Symfony SonataBlockBundle
sonata-project/cache                     2.1.1              2.1.1              Cache library
sonata-project/doctrine-extensions       1.12.0             1.13.0             Doctrine2 behavioral extensions
sonata-project/doctrine-orm-admin-bundle dev-master df32d93 dev-master df32d93 Integrate Doctrine ORM into the SonataAdminBundle
sonata-project/exporter                  2.6.2              2.7.0              Lightweight Exporter library
sonata-project/form-extensions           1.9.0              1.9.0              Symfony form extensions
sonata-project/twig-extensions           1.6.0              1.6.0              Sonata twig extensions
```

</p>
</details>

## Subject

The global search does not return any results, not even empty boxes with `empty_boxes: show`.

When rolling back to the following versions it works again:
```
"sonata-project/admin-bundle": "dev-master#e186a61",
"sonata-project/doctrine-orm-admin-bundle": "dev-master#750ace9",
```

I took a look into recent commits but could not find any changes directly related to the global search.
Does anyone else have this issue?

**Comment by VincentLanglet** [comment]
Thanks for the report.

It's expected with the last commits, since now filter have to implements SearchableFilterInterface.
But the interface is still not released. I'm waiting for this PR to be merged in order to release 3.103 https://github.com/sonata-project/SonataAdminBundle/pull/7308.

Then, persistence bundle will be updated in order to implement SearchableFilterInterface.

**Comment by moostrich** [comment]
@VincentLanglet Alright, thanks for clarifying. Looking forward to v4! :)
